_data-prepper/logs.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: default
+title: Log4j configuration
+nav_order: 12
+---
+
+# Log4j configuration
+
+This section provides the details for configuring Log4j. 
+
+## Logging 
+
+The following describes how Data Prepper performs logging. Data Prepper uses [SLF4J](http://www.slf4j.org/) with a [Log4j 2 binding](http://logging.apache.org/log4j/2.x/log4j-slf4j-impl/). 
+
+<!--- Editorial: Is it safe to remove this first introductory sentence? I'm not sure if it's necessary or helpful to users.--->
```

Comment:
This seems like something we should ask a SME. Is it necessary to understanding what follows? If yes, then we should include it. If no, you can make a decision regarding whether or not to include it based on its potential helpfulness to the reader as complementary information.

Reply:
@natebower Oh, I was specifically referring only to the sentence: "The following describes how Data Prepper performs logging." The subsequent text seems to explain it. Maybe we can change it to something like "Data Prepper performs logging by using [SLF4J](http://www.slf4j.org/) with a [Log4j 2 binding](http://logging.apache.org/log4j/2.x/log4j-slf4j-impl/)." ?